dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -137,26 +137,27 @@ inline void ParseTarFile(std::vector<SampleDesc>& samples_container,
       continue;
     }
 
-    std::string filename, ext;
-    std::tie(filename, ext) = split_name(tar_archive.GetFileName());
+    std::string root_name, ext;
```

Comment:
Why _root_? It looks like some common prefix to everything - which it is not, because we can have multiple tar archives. A name without an extension is usually referred to as base_name / basename.

Reply:
Changed to `basename`